packages/moonstone/VirtualList/VirtualListBase.js

```diff
@@ -130,7 +130,7 @@ const VirtualListBaseFactory = (type) => {
 			 *
 			 * @type {Number}
 			 * @default 0
-			 * @public
+			 * @private
```

Comment:
I'm not sure it should be `private` or `public`...

Reply:
As you mentioned, I reverted from `private` to `public`.